tests/integration/tests.c

```diff
@@ -331,31 +294,31 @@ static void test_graph_type1(void)
 			   0x000000EE, mac_vtep, PG_VTEP_DST_PORT,
 			   PG_VTEP_ALL_OPTI, &error);
 	check_error(error);
-
 	print = pg_print_new("print", NULL, PG_PRINT_FLAG_MAX,
 			     NULL, &error);
 	check_error(error);
-
-	PG_GC_CHAINED_ADD(brick_gc, nic, vtep, print);
-
 	g_assert(add_graph_branch(&branch1, 1, mac1, 0, 0));
 	g_assert(add_graph_branch(&branch2, 2, mac2, 0, 0));
 
 	/*
 	 * Our main graph:
-	 *	                FIREWALL1 -- ANTISPOOF1 -- VHOST1
+	 *	                FIREWALL1 -- VHOST1
```

Comment:
Can you add a todo please ?
we don't use antispoof yet, but we should use them someday, so please add a todo